### PR TITLE
Added light intensity to light.proto

### PR DESCRIPTION
**Pull request by ahcorde**
Added light intensity field to light.proto. Related with this issue https://github.com/ignitionrobotics/ign-rendering/issues/232

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: proto/ignition/msgs/light.proto [hunk 1]
```diff
@@ -55,6 +55,7 @@ message Light
   float spot_inner_angle        = 13;
   float spot_outer_angle        = 14;
   float spot_falloff            = 15;
+  float intensity            = 18;
```
Comment (chapulina) [inline proto/ignition/msgs/light.proto:58]: Can we put it in order below? Also, please document all new fields.

**Review by iche033 — APPROVED** [review]

File: proto/ignition/msgs/light.proto [hunk 2]
```diff
@@ -61,4 +61,7 @@ message Light
 
   /// \brief Unique id of light's parent
   uint32 parent_id              = 17;
+
+  /// \brief light intensity
+  float intensity               = 18;
```
Comment (iche033) [inline proto/ignition/msgs/light.proto:66]: can we default this to `1.0`?

Comment (ahcorde) [reply proto/ignition/msgs/light.proto:66]: Explicit default values are not allowed in proto3.

Comment (ahcorde) [reply proto/ignition/msgs/light.proto:66]: @iche033 can we merge this then?

Comment (ahcorde) [reply proto/ignition/msgs/light.proto:66]: maybe we can force this as a required field

Comment (iche033) [reply proto/ignition/msgs/light.proto:66]: ok I see, lets leave it as is then.

**Pull request merged**